src/main/resources/plasmid.mixins.json

```diff
@@ -11,12 +11,13 @@
     "custom.EntityMixin",
     "custom.ServerPlayNetworkHandlerMixin",
     "event.ExplosionMixin",
+    "event.LivingEntityMixin",
     "event.ServerPlayerEntityMixin",
     "event.ServerPlayerInteractionManagerMixin",
     "map.ServerPlayerEntityMixin",
+    "rule.AreaHelperMixin",
     "rule.CraftingResultInventoryMixin",
     "rule.HungerManagerMixin",
-    "rule.AreaHelperMixin",
```

Comment:
nothing important but why did this get moved?

Reply:
I have no clue. It may have been MCDev doing some sort of auto-formatting?